### PR TITLE
Get rid of get_ prefixes

### LINKED ISSUE #154: Get rid of `get_` prefixes
We should rename APIs like `static_map::get_size()` to `static_map::size()` to align with the common naming practice in libcudf. The `get_` prefix is unnecessary.

### PR DESCRIPTION
**Pull request by vighnesh-kadam**
Thank you for contributing to ___PROJECT___ :)

Here are some guidelines to help the review process go smoothly.

1. Please write a description in this text box of the changes that are being
   made.
Get rid of get_ prefixes
2. Please ensure that you have written units tests for the changes made/features
   added.

3. If you are closing an issue please use one of the automatic closing words as
   noted here: https://help.github.com/articles/closing-issues-using-keywords/
Closes [#1478](https://github.com/NVIDIA/cuCollections/issues/154)
4. If your pull request is not ready for review but you want to make use of the
   continuous integration testing facilities please label it with `[WIP]`.

5. If your pull request is ready to be reviewed without requiring additional
   work on top of it, then remove the `[WIP]` label (if present) and replace
   it with `[REVIEW]`. If assistance is required to complete the functionality,
   for example when the C/C++ code of a feature is complete but Python bindings
   are still required, then add the label `[HELP-REQ]` so that others can triage
   and assist. The additional changes then can be implemented on top of the
   same PR. If the assistance is done by members of the rapidsAI team, then no
   additional actions are required by the creator of the original PR for this,
   otherwise the original author of the PR needs to give permission to the
   person(s) assisting to commit to their personal fork of the project. If that
   doesn't happen then a new PR based on the code of the original PR can be
   opened by the person assisting, which then will be the PR that will be
   merged.

6. Once all work has been done and review has taken place please do not add
   features or make changes out of the scope of those requested by the reviewer
   (doing this just add delays as already reviewed code ends up having to be
   re-reviewed/it is hard to tell what is new etc!). Further, please do not
   rebase your branch on master/force push/rewrite history, doing any of these
   causes the context of any comments made by reviewers to be lost. If
   conflicts occur against master they should be resolved by merging master
   into the branch used for making the pull request.

Many thanks in advance for your cooperation!

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Comment by raydouglass** [comment]
okay to test

**Comment by vighnesh-kadam** [comment]
Can you Please tell me what mistakes i have made and how i can correct them.

**Comment by PointKernel** [comment]
@vighnesh-kadam You renamed all related APIs in `include/cuco` but didn't change the code in `benchmarks`, `examples`, and `tests`. You may want to [build your code locally](https://github.com/NVIDIA/cuCollections#building-cucollections) to catch any leftovers and make sure all code can compile and run.

**Comment by vighnesh-kadam** [comment]
Okay got it ! I will get it done.

**Comment by vighnesh-kadam** [comment]
I am not able to build the repo locally . i keep getting the following error after i run `cmake ..` command :

> CMake Warning:
>   Ignoring extra path from command line:
> 
>    ".."
> CMake Error: The source directory "C:/Users/vighn" does not appear to contain CMakeLists.txt.

**Comment by PointKernel** [comment]
> I am not able to build the repo locally . i keep getting the following error after i run `cmake ..` command :
> 
> > CMake Warning:
> > Ignoring extra path from command line:
> > ".."
> > CMake Error: The source directory "C:/Users/vighn" does not appear to contain CMakeLists.txt.

@vighnesh-kadam  Looks like you are invoking CMake command in the wrong folder. In geneal, you want to create a new folder in `cuCollections` (let's say `cuCollections/build`). Then inside `build`, type `cmake ../` in the command line. You may want to check CMake tutorial (https://cmake.org/cmake/help/latest/guide/tutorial/index.html) also to understand how it works. It seems you are building in Windows as well and I'm not 100% sure that cuCollections can be built flawlessly in windows. Yours to discover...

**Comment by vighnesh-kadam** [comment]
> > I am not able to build the repo locally . i keep getting the following error after i run `cmake ..` command :
> > > CMake Warning:
> > > Ignoring extra path from command line:
> > > ".."
> > > CMake Error: The source directory "C:/Users/vighn" does not appear to contain CMakeLists.txt.
> 
> @vighnesh-kadam Looks like you are invoking CMake command in the wrong folder. In geneal, you want to create a new folder in `cuCollections` (let's say `cuCollections/build`). Then inside `build`, type `cmake ../` in the command line. You may want to check CMake tutorial (https://cmake.org/cmake/help/latest/guide/tutorial/index.html) also to understand how it works. It seems you are building in Windows as well and I'm not 100% sure that cuCollections can be built flawlessly in windows. Yours to discover...

**Comment by vighnesh-kadam** [comment]
ok i will try the build again in ubuntu

**Comment by vighnesh-kadam** [comment]
> > I am not able to build the repo locally . i keep getting the following error after i run `cmake ..` command :
> > > CMake Warning:
> > > Ignoring extra path from command line:
> > > ".."
> > > CMake Error: The source directory "C:/Users/vighn" does not appear to contain CMakeLists.txt.
> 
> @vighnesh-kadam Looks like you are invoking CMake command in the wrong folder. In geneal, you want to create a new folder in `cuCollections` (let's say `cuCollections/build`). Then inside `build`, type `cmake ../` in the command line. You may want to check CMake tutorial (https://cmake.org/cmake/help/latest/guide/tutorial/index.html) also to understand how it works. It seems you are building in Windows as well and I'm not 100% sure that cuCollections can be built flawlessly in windows. Yours to discover...

I was able to install cmake on my ubuntu sys. but the problem is im not able to install cuda toolkit hence im not able to get the nvcc compiler ,  i tried insalling deb and local run from here [toolkit install ](https://developer.nvidia.com/cuda-downloads?target_os=Linux&target_arch=x86_64&Distribution=Ubuntu&target_version=20.04) but failed . 
im still trying other methods , 
Also it is because my system doesn't have a GPU?

**Comment by PointKernel** [comment]
> > > I am not able to build the repo locally . i keep getting the following error after i run `cmake ..` command :
> > > > CMake Warning:
> > > > Ignoring extra path from command line:
> > > > ".."
> > > > CMake Error: The source directory "C:/Users/vighn" does not appear to contain CMakeLists.txt.
> > 
> > 
> > @vighnesh-kadam Looks like you are invoking CMake command in the wrong folder. In geneal, you want to create a new folder in `cuCollections` (let's say `cuCollections/build`). Then inside `build`, type `cmake ../` in the command line. You may want to check CMake tutorial (https://cmake.org/cmake/help/latest/guide/tutorial/index.html) also to understand how it works. It seems you are building in Windows as well and I'm not 100% sure that cuCollections can be built flawlessly in windows. Yours to discover...
> 
> I was able to install cmake on my ubuntu sys. but the problem is im not able to install cuda toolkit hence im not able to get the nvcc compiler , i tried insalling deb and local run from here [toolkit install ](https://developer.nvidia.com/cuda-downloads?target_os=Linux&target_arch=x86_64&Distribution=Ubuntu&target_version=20.04) but failed . im still trying other methods , Also it is because my system doesn't have a GPU?

Have you tried the apt install? i.e.
```bash
sudo apt update
sudo apt install nvidia-cuda-toolkit
```
Otherwise, you can use a docker image to just compile the code. e.g. https://hub.docker.com/r/nvidia/cuda

**Comment by vighnesh-kadam** [comment]
> > > > I am not able to build the repo locally . i keep getting the following error after i run `cmake ..` command :
> > > > > CMake Warning:
> > > > > Ignoring extra path from command line:
> > > > > ".."
> > > > > CMake Error: The source directory "C:/Users/vighn" does not appear to contain CMakeLists.txt.
> > > 
> > > 
> > > @vighnesh-kadam Looks like you are invoking CMake command in the wrong folder. In geneal, you want to create a new folder in `cuCollections` (let's say `cuCollections/build`). Then inside `build`, type `cmake ../` in the command line. You may want to check CMake tutorial (https://cmake.org/cmake/help/latest/guide/tutorial/index.html) also to understand how it works. It seems you are building in Windows as well and I'm not 100% sure that cuCollections can be built flawlessly in windows. Yours to discover...
> > 
> > 
> > I was able to install cmake on my ubuntu sys. but the problem is im not able to install cuda toolkit hence im not able to get the nvcc compiler , i tried insalling deb and local run from here [toolkit install ](https://developer.nvidia.com/cuda-downloads?target_os=Linux&target_arch=x86_64&Distribution=Ubuntu&target_version=20.04) but failed . im still trying other methods , Also it is because my system doesn't have a GPU?
> 
> Have you tried the apt install? i.e.
> 
> ```shell
> sudo apt update
> sudo apt install nvidia-cuda-toolkit
> ```
> 
> Otherwise, you can use a docker image to just compile the code. e.g. https://hub.docker.com/r/nvidia/cuda

yes i tried the sudo apt commands but still got many errors 
 I will do it using docker  now

File: benchmarks/hash_table/static_multimap/count_bench.cu [hunk 1]
```diff
@@ -32,9 +32,9 @@ std::enable_if_t<(sizeof(Key) == sizeof(Value)), void> nvbench_static_multimap_c
   nvbench::state& state,
   nvbench::type_list<Key, Value, nvbench::enum_type<Dist>, nvbench::enum_type<Multiplicity>>)
 {
-  auto const num_keys      = state.get_int64("NumInputs");
-  auto const occupancy     = state.get_float64("Occupancy");
-  auto const matching_rate = state.get_float64("MatchingRate");
+  auto const num_keys      = state.int64("NumInputs");
+  auto const occupancy     = state.float64("Occupancy");
+  auto const matching_rate = state.float64("MatchingRate");
```
Comment (PointKernel) [inline benchmarks/hash_table/static_multimap/count_bench.cu]:
```suggestion
  auto const num_keys      = state.get_int64("NumInputs");
  auto const occupancy     = state.get_float64("Occupancy");
  auto const matching_rate = state.get_float64("MatchingRate");
```

File: benchmarks/hash_table/static_multimap/count_bench.cu [hunk 2]
```diff
@@ -61,7 +61,7 @@ std::enable_if_t<(sizeof(Key) == sizeof(Value)), void> nvbench_static_multimap_c
   map.insert(d_pairs.begin(), d_pairs.end());
 
   state.exec(nvbench::exec_tag::sync, [&](nvbench::launch& launch) {
-    auto count = map.count(d_keys.begin(), d_keys.end(), launch.get_stream());
+    auto count = map.count(d_keys.begin(), d_keys.end(), launch.stream());
```
Comment (PointKernel) [inline benchmarks/hash_table/static_multimap/count_bench.cu]:
```suggestion
    auto count = map.count(d_keys.begin(), d_keys.end(), launch.get_stream());
```

File: benchmarks/analysis/notebooks/Utils.py [hunk 3]
```diff
@@ -17,7 +17,7 @@ def plot_single_perf(bm, df, xaxis, unique_labels):
     
     ax.set_xscale('log')
     ax.set_xticks(list(df[xaxis]))
-    ax.get_xaxis().set_major_formatter(matplotlib.ticker.ScalarFormatter())
+    ax.xaxis().set_major_formatter(matplotlib.ticker.ScalarFormatter())
```
Comment (PointKernel) [inline benchmarks/analysis/notebooks/Utils.py]:
```suggestion
    ax.get_xaxis().set_major_formatter(matplotlib.ticker.ScalarFormatter())
```

**Comment by sleeepyjack** [comment]
Closing this issue as it's stale and will be implicitly fixed by #110

**Pull request closed without merging**